nixos/modules/services/system/userborn.nix

```
@@ -111,6 +143,7 @@ in
           );
 
       services.userborn = {
```

We shouldn't even include it then:

```suggestion
      services.userborn = lib.mkIf (!cfg.static) {
```